packages/blade/src/components/Card/Divider.native.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components/native';
+import { makeSize } from '~utils';
+
+const Divider = styled.View(({ theme }) => {
```

Comment:
use box here? and then you might then need just one divider file?

Reply:
Yeah I tried that earlier, it was making the Box component quite complex. Complex in terms of adding other border support like borderTop, borderBottom, borderLeft. so just added this here only.

Reply:
can you post a sample snippet of complexity it is creating?

Reply:
The main factor is rn. 

rn doesn't support shorthand form of the border properties so have to use the `borderBottomWidth`, I can add it in just for bottom but later on if we try to add more there will be a lot of checks needed like this: 

```
  ...(borderBottomWidth && { borderBottomStyle: 'solid', borderBottomWidth: 1 }),
  ...(borderTopWidth && { borderTopStyle: 'solid', borderTopWidth: 1 }),
```

Don't want to implement a half backed API in the Box, we are already using it everywhere on the codebase.